test/unit/middlewares/user-validator.test.js

```diff
@@ -308,6 +308,56 @@ describe("Middleware | Validators | User", function () {
     });
   });
 
+  describe("Test the update Self Validator for disabled_roles property", function () {
+    it("Allows the request to pass with disabled_roles property", async function () {
+      const req = {
+        body: {
+          disabledRoles: [],
+        },
+      };
+      const res = {};
+      const nextSpy = sinon.spy();
+      await updateUser(req, res, nextSpy);
+      expect(nextSpy.callCount).to.be.equal(1);
+    });
+
+    it("Allows the request to pass with disabled_roles property with roles `super_user` & `member'", async function () {
+      const req = {
+        body: {
+          disabledRoles: ["super_user", "member"],
+        },
+      };
+      const res = {};
+      const nextSpy = sinon.spy();
+      await updateUser(req, res, nextSpy);
+      expect(nextSpy.notCalled).to.be.equal(false);
```

Comment:
Please update it to the callCount, because here expecting a negative test case is confusing